lib/order.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'takeaway'
+
+class Order
+
+  attr_reader :check_total
+  attr_reader :pay
+  attr_reader :place
+
+  def initialize
+  end
+
+  def pay price
```

Comment:
Unused method argument - `price`. If it's necessary, use `_` or `_price` as an argument name to indicate that it won't be used. You can also write as `pay(*)` if you want the method to accept any arguments but don't care about them.